packages/insight/src/components/pill.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+import CloseLightSvg from 'src/assets/images/close-light.svg'
+import {Black, Slate30} from '../assets/styles/colors';
+
+const PillBubble = styled.div`
+  padding: 7px;
+  padding-right: 10px;
```

Comment:
Please remove this. It adds extra spacing to the pill on the inside right. The `padding: 7px;` on L7 by itself creates a uniform padding around the pill.
<img width="1225" alt="Screenshot 2025-05-27 at 12 44 28 PM" src="https://github.com/user-atta
<img width="1218" alt="Screenshot 2025-05-27 at 12 44 45 PM" src="https://github.com/user-attachments/assets/00c50bc6-633b-4dc4-bf77-8a95750080ba" />
chments/assets/10edc5d1-a7ce-42c0-82cd-a7e943a93038" />